app/helpers/forever_style_guide/application_helper.rb

```diff
@@ -9,9 +9,13 @@ def trademark(copy)
         copy.upcase
       when "Forever Historian", "Forever Valet", "Forever Live!", "Forever Print"
         copy[0..6].upcase + copy[7..copy.length-1] + "™"
-      when "pixels2Pages", "Historian"
+      when "pixels2Pages", "Historian", "ClickCollage", "It's like insurance for your photos", "Paper Tray Studio", "p2P", "StoryBook Legacy", "Templates & Co", "Valet"
         copy + "™"
-      when "Forever Account", "Forever Guarantee", "Forever Guarantee Fund", "Forever Membership", "Forever Guarantee Fund", "Forever Members", "Forever Retreats", "Forever Ambassador", "Forever Ambassadors"
+      when "Forever Account", "Forever Guarantee", "Forever Guarantee Fund", "Forever Membership", "Forever Guarantee Fund",
```

Comment:
Forever Guarantee Fund is listed twice don't think it hurts but should prob clean it up